File: eTeatar/eTeatar.Model/KorisnikUloga.cs
using System;
using System.Collections.Generic;

namespace eTeatar.Model
{
    public partial class KorisnikUloga
    {
        public int KorisnikUlogaId { get; set; }

        public int KorisnikId { get; set; }

        public int UlogaId { get; set; }

        public virtual Uloga Uloga { get; set; }
    }
}

File: eTeatar/UI/eteatar_mobile/lib/models/korisnik_uloga.dart

import 'package:eteatar_mobile/models/uloga.dart';
import 'package:json_annotation/json_annotation.dart';

part 'korisnik_uloga.g.dart';

@JsonSerializable()
class KorisnikUloga {
  int? korisnikUlogaId;
  int? korisnikId;
  int? ulogaId;
  Uloga? uloga;
  KorisnikUloga(
    this.korisnikUlogaId,
    this.korisnikId,
    this.ulogaId,
    this.uloga,
  );

  factory KorisnikUloga.fromJson(Map<String, dynamic> json) => _$KorisnikUlogaFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KorisnikUlogaToJson(this);
}